templates/account.html

```diff
@@ -30,14 +32,15 @@ <h5>{{game_name}}</h5>
     {% endfor %}
     {% endif %}
     {% endwith %}
+
     <br>
     <br>
-    <form method="POST" action="/account">
+    <form method="POST" action="/save_game">
         <input type="text" name="game_name" placeholder="Add Games!" />
         <input type="submit" class="submit" value="Submit" />
     </form>
     <br>
-    <form method="POST" action="/main">
+    <form method="POST" action="/main2">
```

Comment:
Nice catch this was causing us problems with exiting the account page 
maybe rename the two routes so that we wont have the same problem again 
something to the effect of /main for the main page and /gatherinfo to get all the game data that needs to be displayed